test/validators/activity.unit.test.js

```diff
@@ -0,0 +1,180 @@
+const activityValidator = require('../../src/validators/activity')
+const ActivitiesApi = require('../../src/api/activities')
+const Common = require('../../src/validators/common')
+
+jest.mock('../../src/api/activities')
+jest.mock('../../src/validators/common')
+
+describe('activity.unit', () => {
+  beforeEach(() => {
+    jest.resetModules()
+  })
+
+  const getMockRequest = (
+    cacheObj = {
+      submissionId: 'sub-1'
+    },
+    payload = {
+      river: 'r1',
+      daysFishedOther: 1,
+      daysFishedWithMandatoryRelease: 2
+    }) => {
+    const cache = {
+      get: jest.fn().mockResolvedValue(cacheObj)
+    }
+
+    return {
+      payload,
+      cache: jest.fn(() => cache)
+    }
+  }
+
+  const setupCommonMocks = ({
+    daysOther = 1,
+    daysMandatory = 2
+  } = {}) => {
```

Comment:
I don't think these arguments are ever overwritten, so we probably don't need them.